Duplicati/Library/Backend/Tardigrade/TardigradeBackend.cs

```diff
@@ -59,6 +80,9 @@ public Tardigrade()
         // This constructor is needed by the BackendLoader.
         public Tardigrade(string url, Dictionary<string, string> options)
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I like the use of `RuntimeInformation.IsOSPlatform`, but we should probably be consistent and use `Duplicati.Library.Common.Platform.IsClientWindows`.  We can consider changing the implementation of `Platform.IsClientWindows` in the future.

Reply:
No problem, I've changed it.